workers/generic-worker/usage.go

```diff
@@ -32,15 +32,19 @@ and reports back results to the queue.
 
   Usage:
     generic-worker run                      [--config         CONFIG-FILE]
+                                            [--with-worker-runner]
                                             [--worker-runner-protocol-pipe PIPE]
                                             [--configure-for-aws | --configure-for-gcp | --configure-for-azure]` + installServiceSummary() + `
+
     generic-worker show-payload-schema
     generic-worker new-ed25519-keypair      --file ED25519-PRIVATE-KEY-FILE` + customTargetsSummary() + `
     generic-worker --help
     generic-worker --version
 
   Targets:
-    run                                     Runs the generic-worker.
+    run                                     Runs the generic-worker.  Pass --with-worker-runner if
+											running under that service, otherwise generic-worker will
+											not communicate with worker-runner.
```

Comment:
Something gone awry with the formatting here.

Reply:
Hm, it's just tabs vs. spaces.  I thought Go preferred tabs?